core/decs.h

```diff
@@ -368,6 +367,30 @@ extern grid_radtype_type Nem_phys, Nabs_phys, radtype_buf;
 extern grid_int_type    Nsuper;
 extern grid_double_type Esuper;
 extern grid_prim_type   psupersave;
+
+#if LOCAL_ANGULAR_DISTRIBUTIONS
+#define LOCAL_NUM_BASES (2)
+#define MOMENTS_A (0)
+#define MOMENTS_B (1)
+#define MOMENTS_DIFF (2)
+typedef double grid_local_angles_type[LOCAL_NUM_BASES][LOCAL_ANGLES_NX1]
+                                     [LOCAL_ANGLES_NX2][RAD_NUM_TYPES]
+                                     [LOCAL_ANGLES_NMU];
+extern grid_local_angles_type local_angles;
+extern double                 local_dx1_rad, local_dx2_rad, local_dx_costh;
+
+#if RAD_NUM_TYPES >= 4
+typedef double grid_Gnu_type[LOCAL_NUM_BASES][LOCAL_ANGLES_NX1]
+                            [LOCAL_ANGLES_NX2][LOCAL_ANGLES_NMU];
+typedef double grid_local_moment_type[LOCAL_NUM_BASES][3][LOCAL_ANGLES_NX1]
```

Comment:
What does the [3] do here (line 385)?

Reply:
I should potentially hardcode that to a named variable `LOCAL_NUM_MOMENTS` but it's basically `A`, `B` or `A - B`

Reply:
Renamed